cloudflare-rails.gemspec

```diff
@@ -18,25 +18,25 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
 
   spec.add_development_dependency 'appraisal', '~> 2.5.0'
-  spec.add_development_dependency 'bundler', '>= 2.1.2'
+  spec.add_development_dependency 'bundler', '>= 2.4.18'
   spec.add_development_dependency 'pry-byebug'
   spec.add_development_dependency 'rack-attack', '~> 6.7.0'
-  spec.add_development_dependency 'rake', '~> 13.1.0'
-  spec.add_development_dependency 'rspec', '~> 3.12.0'
+  spec.add_development_dependency 'rake', '~> 13.2.1'
+  spec.add_development_dependency 'rspec', '~> 3.13.0'
   spec.add_development_dependency 'rspec_junit_formatter', '~> 0.6.0'
-  spec.add_development_dependency 'rspec-rails', '~> 6.1.0'
-  spec.add_development_dependency 'rubocop', '~> 1.60.2'
-  spec.add_development_dependency 'rubocop-performance', '~> 1.20.2'
-  spec.add_development_dependency 'rubocop-rails', '~> 2.23.1'
+  spec.add_development_dependency 'rspec-rails', '~> 6.1.2'
+  spec.add_development_dependency 'rubocop', '~> 1.64.1'
+  spec.add_development_dependency 'rubocop-performance', '~> 1.21.0'
+  spec.add_development_dependency 'rubocop-rails', '~> 2.25.0'
   spec.add_development_dependency 'rubocop-rspec', '~> 2.26.1'
-  spec.add_development_dependency 'webmock', '~> 3.19.1'
+  spec.add_development_dependency 'webmock', '~> 3.23.1'
 
-  spec.add_dependency 'actionpack', '>= 6.1', '< 7.2.0'
-  spec.add_dependency 'activesupport', '>= 6.1', '< 7.2.0'
-  spec.add_dependency 'railties', '>= 6.1', '< 7.2.0'
+  spec.add_dependency 'actionpack', '>= 7.1.0', '< 8.1.0'
+  spec.add_dependency 'activesupport', '>= 7.1.0', '< 8.1.0'
+  spec.add_dependency 'railties', '>= 7.1.0', '< 8.1.0'
```

Comment:
I do understand the lower bound limit requirement.

However making an upper bound limit will always make it a hassle for people running on edge or unreleased version. (or recently released version, but you forgot to bump the dependency)

And even if let's say 8.1.0 gets released, people will be blocked by your gem to update to rails, forcing them to :
- fork the repo and remove this upper bound
- OR if forking is not a possibility, simply copy paste the code of this gem and include it in their app

My point is, since you have an appraisal that targets `rails main/edge`, you'll have a lot of opportunities when you release patch/minor fixes to **catch bugs earlier** in Github CI.
==> making your gem more future proof
==> and removing the need for this limiting upper bound